README.md

```diff
@@ -17,31 +16,43 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
 
 ## Table of Contents
 - [Usage](#usage)
-- [API](#api)
   - [Blocks API](#blocks-api)
     - [Configuration](#configuration)
   - [Styles API](#styles-api)
 - [Troubleshooting](#troubleshooting)
 
 ## Usage
-:construction: :construction: :construction:
 
-To use this app you need to import in your dependencies on `manifest.json`.
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
+
+To use this app, you need to import in your dependencies on `manifest.json`.
```

Comment:
:hammer:  ... import in your `dependencies` on the `manifest.json` file.

Reply:
:hammer:  To use this app, you need to add it in the `dependencies` on `manifest.json` file.